README.md

```diff
@@ -6,10 +6,83 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an API for operating and monitoring electrical
+components and sensors within a microgrid. It offers various functionalities
+related to microgrid management, including metadata retrieval, component
+listing, setting power levels, and controlling the state of components. The API
+is structured around the concept of a microgrid, which is a localized energy
+system composed of various interconnected components and auxiliary sensors.
```

Comment:
Shall me make a distinction between island and grid connected microgrids?

Reply:
I added one line there mentioning this.